anxcloud/schema_lbaas.go

```diff
@@ -0,0 +1,18 @@
+package anxcloud
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func schemaLBaaSLoadbalancer() map[string]*schema.Schema {
```

Comment:
we should probably stick to one writing style:
* Load**b**alancer
* Load**B**alancer - I prefer this one